abis_mapping/templates/survey_site_visit_data_v2/mapping.py

```diff
@@ -112,6 +112,37 @@ def apply_validation(
         # Return validation report
         return report
 
+    def extract_site_id_map(
```

Comment:
"site_id_map" is already a thing from one of the other extraction methods.
Can this be `extract_site_visit_id_to_site_id_map` to avoid confusion

Reply:
done.